packages/paste-core/components/textarea/stories/textarea.stories.tsx

```diff
@@ -59,6 +63,31 @@ TextareaInverse.story = {
   name: 'Textarea - inverse',
 };
 
+export const TextareaResizeVertical = (): React.ReactNode => {
```

Comment:
thought(blocking): can you add a story similar to our codesandbox, where we add and remove multiple textareas? This will help us verify resize continues to work with several of them mounted